### ISSUE TITLE
Loading page switch

### ISSUE THREAD

**Issue by lenisko** [chao/RESTClient#272]
Please add support to disable this loading window. 
I don't want to offend anyone, it's fancy.. but when someone is working with many URLs this blinking it's driving nuts. 
Some switch to enable/disable this would be great, in disabled version counter would be great to have :-)

![2017-11-30_03-07-26-firefox](https://user-images.githubusercontent.com/10072920/33409432-a4eb8c3a-d57b-11e7-8f3b-f65bedb4df8e.jpg)

**Comment by chao** [comment]
You are right, I'm going to implement this feature after I finished the oAuth authentication.
